### PR TITLE
Fix bug in the printing of double for limit values

### PR DESCRIPTION
**Pull request by romainbrenguier**
This corrects a bug in which the minimal value for a double was not printed precisely enough.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [na] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [na] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: jbmc/regression/jbmc/double1/test-limit.desc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CORE
+double1.class
+ --function double1.testLimit --trace
+^EXIT=0$
```
Comment (tautschnig) [inline jbmc/regression/jbmc/double1/test-limit.desc]: I think this should be `EXIT=10`

File: jbmc/regression/jbmc/double1/test-limit.desc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+^SIGNAL=0$
+^VERIFICATION FAILED$
+INPUT arg0d: 0x1p-1022 /\* 2.22507e-308
+--
+^warning: ignoring
+--
+Test double value are printed with sufficient precision
```
Comment (tautschnig) [inline jbmc/regression/jbmc/double1/test-limit.desc]: GitHub would like to have a newline here.

**Review by tautschnig — APPROVED** [review]
CI is failing, hopefully just because of the EXIT=0 noted below.

**Review by smowton — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: f8ef55a33d0b64ac5729498145dbb5346fe8bffb).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/107499837
Status will be re-evaluated on next push.
Common spurious failures include: the cbmc commit has disappeared in the mean time (e.g. in a force-push); the author is not in the list of contributors (e.g. first-time contributors); compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: f99235290f47f84907bbffb4c51ab92ca7b6d09f).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/107534355
Status will be re-evaluated on next push.
Common spurious failures include: the cbmc commit has disappeared in the mean time (e.g. in a force-push); the author is not in the list of contributors (e.g. first-time contributors); compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: b453805896b72f7ed0a9ff5ea050b80070662f52).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/107581083
Status will be re-evaluated on next push.
Common spurious failures include: the cbmc commit has disappeared in the mean time (e.g. in a force-push); the author is not in the list of contributors (e.g. first-time contributors); compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 09801c3249d65b0e621197db89d3a89d2c9f4796).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/107586414
Status will be re-evaluated on next push.
Common spurious failures include: the cbmc commit has disappeared in the mean time (e.g. in a force-push); the author is not in the list of contributors (e.g. first-time contributors); compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 9acc29937157f1e87543f5a18a9dddc110042aa0).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/107590471
Status will be re-evaluated on next push.
Common spurious failures include: the cbmc commit has disappeared in the mean time (e.g. in a force-push); the author is not in the list of contributors (e.g. first-time contributors); compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: edfccab4cff58c82698e45218a442e9c532aa0b5).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/107595723
Status will be re-evaluated on next push.
Common spurious failures include: the cbmc commit has disappeared in the mean time (e.g. in a force-push); the author is not in the list of contributors (e.g. first-time contributors); compatibility was already broken by an earlier merge.

**Pull request merged**